File: VB/MvcApp_Q367963/Controllers/HomeController.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Web
Imports System.Web.Mvc
Imports System.Data
Imports DevExpress.Web.Mvc

Namespace MvcApp_Q367963
    Public Class HomeController
        Inherits Controller

        Public Function Index() As ActionResult
            ViewBag.Message = "Welcome to DevExpress Extensions for ASP.NET MVC!"
            ViewData("ComboBoxValue") = 5
            Return View()
        End Function

        Public Function ComboBoxPartial() As ActionResult
            Return PartialView("_PartialPage1")
        End Function
    End Class
End Namespace

File: CS/MvcApp_Q367963/Controllers/HomeController.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.Mvc;
using System.Data;
using DevExpress.Web.Mvc;

namespace MvcApp_Q367963 {
    public class HomeController : Controller {
        public ActionResult Index() {
            ViewBag.Message = "Welcome to DevExpress Extensions for ASP.NET MVC!";
            ViewData["ComboBoxValue"] = 5;
            return View();
        }

        public ActionResult ComboBoxPartial() {
            return PartialView("_PartialPage1");
        }
    }
}
